tests/baselines/reference/tsserver/applyChangesToOpenFiles/with-applyChangedToOpenFiles-request.js

```diff
@@ -124,7 +124,9 @@ Info seq  [hh:mm:ss:mss] event:
             "deferred": 0,
             "deferredSize": 0
           },
-          "compilerOptions": {},
+          "compilerOptions": {
+            "allowNonTsExtensions": true
```

Comment:
The LS sets `allowNonTsExtensions` for any project with js and any inferred project forcibly - this wasn't reflected in telemetry (and _all these baselines_), since without an options declaration, the reverse-serializer couldn't serialize the option.